### PR TITLE
Differenciate now and compensated_now.

### LINKED ISSUE #1907: Differentiate UTime::now() from UTime::now_with_compensation(i64)
The situation, for the moment we have this `now` function that take a compensation. May be we would like a `now` function (without argument), `now_with_compensation` (with a compensation) and a `now_with_offset` that is basically the same of the previous one but with a more adapted name in this case?

If you want I can create a _good first issue_, I would prefer to do this in an other PR later. :pray:

_Originally posted by @adrien-zinger in https://github.com/massalabs/massa/pull/1845#discussion_r761782959_

### PR DESCRIPTION
**Pull request by AureliaDolo**
Fix #1907

### CODE REVIEW

**Review by yvan-sraka — DISMISSED** [review]
LGTM! Maybe we could write some internal doc explaining the concept of "compensation"? or it could be a crate-level doc comment?

**Comment by AureliaDolo** [comment]
Don't you think that the doc comment of compensated_now is enough ?

**Comment by yvan-sraka** [comment]
bors merge

**Comment by AureliaDolo** [comment]
bors merge

**Comment by AureliaDolo** [comment]
Doctests are failing here but not locally, it seems like the doctests here are note happy with     `/// # use massa_time::*;` 
@yvan-sraka have you any hints on that ?

**Comment by yvan-sraka** [comment]
> Doctests are failing here but not locally, it seems like the doctests here are note happy with `/// # use massa_time::*;` @yvan-sraka have you any hints on that?

The issue is more linked with the test that try to stop `massa-node` gracefully https://github.com/massalabs/massa/runs/4467814181?check_suite_focus=true#step:6:176

**Review by yvan-sraka — DISMISSED** [review]

**Comment by yvan-sraka** [comment]
bors merge

File: massa-time/src/lib.rs [hunk 1]
```diff
@@ -127,6 +127,28 @@ impl UTime {
         Ok(UTime(compensated))
     }
 
+    /// Gets current unix timestamp (resolution: milliseconds).
+    ///
+    /// ```
+    /// # use std::time::{Duration, SystemTime, UNIX_EPOCH};
+    /// # use massa_time::*;
```
Comment (yvan-sraka) [inline massa-time/src/lib.rs:134]: @AureliaDolo here I fixed doctests :)

**Review by yvan-sraka — DISMISSED** [review]

**Comment by yvan-sraka** [comment]
bors merge

**Review by yvan-sraka — APPROVED** [review]

**Comment by yvan-sraka** [comment]
bors merge

**Review by yvan-sraka — APPROVED** [review]

**Comment by yvan-sraka** [comment]
bors merge

**Comment by yvan-sraka** [comment]
bors merge

**Comment by AureliaDolo** [comment]
@yvan-sraka I don't understand why it fails :confused:

**Comment by AureliaDolo** [comment]
bors merge

**Comment by yvan-sraka** [comment]
bors merge

**Pull request merged**